### DISCUSSION TITLE
Multiple Snort rule feeds

### DISCUSSION THREAD

**Discussion by tomjones1977** [Security-Onion-Solutions/securityonion#6475] [General] [answered]
Is it possible to configure the global.sls file to allow multiple feeds, such as ET and TALOS at the same time?

**Comment by dougburks** [comment, marked as answer]
Based on the [Other section of the Rules page](https://docs.securityonion.net/en/2.3/rules.html#other), I think you could make this work. However, most organizations find multiple rulesets to be overwhelming unless they are very heavily tuned for their environment.
